api/src/controllers/inbox.controller.ts

```diff
@@ -22,6 +22,11 @@ class InboxController {
   }
   
   public async adminCheck(placeId, id, type): Promise<any> {
+    try {
```

Comment:
Can we add an if typeof type is undefined for all of these new tries?  The reason is if type is colony, hood, or block then the getAdminInfo will run twice.

Reply:
Fixed and ready for review.